pyswarms/backend/handlers.py

```diff
@@ -196,13 +196,15 @@ def reflective(self, position, bounds, **kwargs):
         new_pos = position
         while lower_than_bound[0].size != 0 or greater_than_bound[0].size != 0:
             if lower_than_bound[0].size > 0:
-                new_pos[lower_than_bound] = 2 * lb[lower_than_bound[1]] - \
-                new_pos[lower_than_bound]
+                new_pos[lower_than_bound] = (
```

Comment:
Please remove the formatting changes. It is true that the code needs some reformatting but it doesn't belong to a bug fix commit. It also adds a lot of noise in the commit such that it is hard to see what were actual changes.